codetransformer/transformers/literals.py

```diff
@@ -42,3 +42,54 @@ def visit_STORE_MAP(self, instr):
 
         yield Instruction(ops.STORE_SUBSCR)
         # TOS  = m
+
+
+class OverloadedLiteralTransformer(CodeTransformer):
+
+    def __init__(self, f):
+        self.f = f
+        super().__init__()
+
+    def visit_consts(self, consts):
+        return super().visit_consts(
+            tuple(
+                # This is all one expression.
+                frozenset(self.visit_consts(tuple(const)))
+                if isinstance(const, frozenset)
+                else self.visit_consts(const)
+                if isinstance(const, tuple)
+                else self.f(const)
+                if isinstance(const, self._type)
+                else const
+                ###
+                for const in consts
+            )
+        )
+
+
+class overloaded_bytes(OverloadedLiteralTransformer):
+    """
+    Decorator that applies a callable to each bytes literal in the decorated
```

Comment:
Don't refer to these as 'decorators' in the docstring, a very valid use case is to transform the code of a module. I would say 'transformer' instead.